### PR TITLE
Adding an optional parameter for offer CRUD

### PR DESCRIPTION
**Pull request by olrakovs-zz**
Adding new optional parameter for existing API

### CODE REVIEW

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj [hunk 1]
```diff
@@ -6,9 +6,9 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Marketplace</PackageId>
     <Description>Microsoft Azure Marketplace Library</Description>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```
Comment (isra-fel) [inline sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj:9]: For feature addition we suggest incrementing the minor version. The patch version is to be updated only when  bug fixing.
```suggestion
    <Version>1.1.0</Version>
```

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj [hunk 2]
```diff
@@ -6,9 +6,9 @@
...
     <AssemblyName>Microsoft.Azure.Management.Marketplace</AssemblyName>
-    <PackageTags>management;marketplace;</PackageTags>
+    <PackageTags>management;marketplace;privatestore</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
         This is a public release of the Azure Marketplace SDK. Included with this release is the ability to create and manage private stores in Marketplace.
```
Comment (isra-fel) [inline sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj:14]: Besides, please update the release notes. Thanks

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Marketplace Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Marketplace.")]
+
+[assembly: AssemblyVersion("1.0.0")]
+[assembly: AssemblyFileVersion("1.0.2")]
```
Comment (isra-fel) [inline sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Properties/AssemblyInfo.cs:11]: Here too
```suggestion
[assembly: AssemblyFileVersion("1.1.0")]
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/tests/Microsoft.Azure.Management.Marketplace.Tests.csproj [hunk 4]
```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Marketplace" Version="1.0.1" />
+    <PackageReference Include="Microsoft.Azure.Management.Marketplace" Version="1.0.2" />
```
Comment (isra-fel) [inline sdk/marketplace/Microsoft.Azure.Management.Marketplace/tests/Microsoft.Azure.Management.Marketplace.Tests.csproj:12]: Your test project should depend on the local SDK project, not public package, so please remove this line.
```suggestion
```

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (olrakovs) [reply sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Properties/AssemblyInfo.cs]: Was changed

File: sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj (hunk 2 above)
Comment (olrakovs) [reply sdk/marketplace/Microsoft.Azure.Management.Marketplace/src/Microsoft.Azure.Management.Marketplace.csproj]: Release was updated

**Comment by olrakovs** [comment]
Thank you @isra-fel, changes were made

**Review by isra-fel — APPROVED** [review]
LGTM

**Review by erich-wang — APPROVED** [review]

**Pull request merged**